tools/findFilesThatContainImages.sh

```diff
@@ -0,0 +1,26 @@
+doit(){
+total_count="$(wc -l < "./imgs.txt")";    
+current_count=1;
+spin='←↖↑↗→↘↓↙'
+i=0;
+tput civis
+while read F; do
+    echo "Checking" $F "("$current_count "of" $total_count "image files)";
```

Comment:
You don't need any of these quotes (`$` variables do get expanded in double quoted strings)
```suggestion
    echo "Checking $F ($current_count of $total_count image files)";
```